packages/electrode-archetype-react-app/README.md

```diff
@@ -236,6 +236,35 @@ var supports = require("electrode-archetype-react-app/supports");
 
 supports.cssModuleHook();
 ```
+## Extending Webpack Configuration
+
+The webpack config for your app is being generated by the archetype. While composing the webpack config, the archetype looks for the `archetype/config` folder at the project root of your app.
+In order to override or extend the webpack configuration you should create a file under `archetype/config/webpack`.
```

Comment:
I think this change might have occurred around the same time, the way mentioned in the comment should still work, but this is the way for extending your webpack config, https://github.com/electrode-io/electrode/blob/master/packages/electrode-archetype-react-app-dev/config/webpack/util/generate-config.js#L15